tgui/packages/tgui/interfaces/Keybinds/Keybind.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+import { Button, LabeledList } from 'tgui-core/components';
+import { isEscape } from 'tgui-core/keys';
+import { BooleanLike } from 'tgui-core/react';
+
+import { useBackend } from '../../backend';
+import { formatKeyboardEvent, isStandardKey } from './formatKeyboardEvent';
+import { KeybindData, KeybindsData } from './types';
+
+interface KeybindProps {
+  keybind: KeybindData;
+  isFocused: BooleanLike;
```

Comment:
```suggestion
  isFocused: boolean;
```
`BooleanLike` is for when it's coming from the backend data, we're actually setting this prop to `true`/`false` so it's an actual `boolean`.